docs/platform/what-is-tizen/versions/tizen-5-5-m2.md

```diff
@@ -0,0 +1,71 @@
+# Tizen 5.5
+
+[The second milestone (M2) release of Tizen 5.5](../../release-notes/tizen-5-5-m2.md) was announced in October 2019.
+Tizen is an open-source operating system (OS) maintained by Samsung and is developed and optimized for smart devices since 2012.
+Currently, more than 130+ million Tizen-based devices such as TVs, watches, Family-Hubs, IoT based home appliances, and so on are commercialised.
+The timeline for commercialisation of the various Samsung devices with Tizen OS is as follows:
```

Comment:
As suggested by @mijin85cho 
We do not necessarily need a list, so kindly update like this: 

The following Samsung devices are commercialised with Tizen OS:

Samsung TVs (launched in 2015), Galaxy Wearables (launched in 2014), Family-Hub Refrigerators (launched in 2016), Smart Signage (launched in 2019), Air Conditioners (launched in 2019), Flip board (launched in 2019) and so on.

Reply:
ok